app/controllers/posts_controller.rb

```diff
@@ -10,7 +10,23 @@ def show
     @post = Post.where(id: params[:id]).first
   end
 
+  def new
+    puts params
```

Comment:
- It's not recommended to print values with production code for security reasons. Please remove those puts statements from your code.